### PR TITLE
Improve crictl UX

### PR DESCRIPTION
**Pull request by feiskyer**

### CODE REVIEW

**Comment by feiskyer** [comment]
cc/ @xlgao-zju

File: README.md [hunk 1]
```diff
@@ -109,43 +109,65 @@ CLI for kubelet CRI
...
+  },
+  "image":{
+      "image": "busybox"
+  },
+  "command": [
+      "top"
+  ],
+  "linux": {
+  }
+}
```
Comment (xlgao-zju) [inline README.md]: Add `Stdin=true`?

File: cmd/crictl/container.go [hunk 2]
```diff
@@ -302,16 +332,24 @@ func ContainerStatus(client pb.RuntimeServiceClient, ID string) error {
 		if r.Status.Metadata.Name != "" {
```
Comment (xlgao-zju) [inline cmd/crictl/container.go]: Do we need to print other fields of `Metadata`?

File: cmd/crictl/image.go [hunk 3]
```diff
@@ -27,22 +27,31 @@ import (
...
-	After:  closeConnection,
+	After: closeConnection,
 }
 
 var pullImageCommand = cli.Command{
-	Name:  "pull",
-	Usage: "pull an image",
+	Name:      "pull",
+	Usage:     "pull an image",
+	ArgsUsage: "IMAGE",
```
Comment (xlgao-zju) [inline cmd/crictl/image.go]: s/IMAGE/[IMAGE]|[IMAGE:TAG]|[IMAGE:DIGEST]

File: cmd/crictl/portforward.go [hunk 4]
```diff
@@ -33,13 +33,19 @@ import (
 )
 
 var runtimePortForwardCommand = cli.Command{
-	Name:  "portforward",
-	Usage: "forword ports from a sandbox",
+	Name:      "portforward",
+	Usage:     "forword ports from a sandbox",
+	ArgsUsage: "SandboxID port",
```
Comment (xlgao-zju) [inline cmd/crictl/portforward.go]: s/port/hostport:containerport

**Comment by xlgao-zju** [comment]
@feiskyer LGTM with few nits

File: README.md (hunk 1 above)
Comment (feiskyer) [inline README.md]: There are much more fields in container config, this is just a simple example. We don't need to add all fields in it.

File: cmd/crictl/container.go (hunk 2 above)
Comment (feiskyer) [inline cmd/crictl/container.go]: What do you mean by other fields? there are only name and attempt in ContainerMetadata.

**Comment by feiskyer** [comment]
@xlgao-zju Updated. PTAL.

**Comment by xlgao-zju** [comment]
LGTM

**Pull request merged**